src/test/java/PracticingTDD/ConnectionManagerTest.java

```diff
@@ -0,0 +1,41 @@
+package PracticingTDD;
```

Comment:
Usually your tests should be in the same declared package as the class being tested.